pkgs/development/ocaml-modules/owee/default.nix

```diff
@@ -1,15 +1,16 @@
-{ lib, buildDunePackage, fetchFromGitHub }:
+{ lib, buildDunePackage, fetchurl, pkg-config }:
 
 buildDunePackage rec {
   minimumOCamlVersion = "4.06";
   pname = "owee";
-  version = "0.3";
+  version = "0.4";
+  useDune2 = true;
 
-  src = fetchFromGitHub {
-    owner = "let-def";
-    repo = "owee";
-    rev = "v${version}";
-    sha256 = "0jp8ca57488d7sj2nqy4yxcdpda6sxx51yyi8k6888hbinhyqp0j";
+  nativeBuildInputs = [ pkg-config ];
+
+  src = fetchurl {
+    url = "https://github.com/let-def/owee/releases/download/v0.4/owee-0.4.tbz";
```

Comment:
```suggestion
    url = "https://github.com/let-def/owee/releases/download/v${version}/owee-${version}.tbz";
```

Reply:
@vbgl did this change in another PR so I dropped it form here